app/presenters/description_display_formatter.rb

```diff
@@ -1,26 +1,32 @@
 require 'rails_autolink'
 # Takes a description string (from a work.description) and formats it for display.
 # Ported from chf-sufia/app/helpers/description_formatter_helper.rb
-
-# DescriptionDisplayFormatter.new(work.description).format
-# DescriptionDisplayFormatter.new(work.description, truncate:true).format
-
+#
+#     DescriptionDisplayFormatter.new(work.description).format
+#     DescriptionDisplayFormatter.new(work.description, truncate:true).format
+#
+# Or for a plain-text description, with html tags stripped:
+#     DescriptionDisplayFormatter.new(work.description, truncate:true).format_plain
```

Comment:
Interesting -- two different output methods on the same formatter object. I kinda like this better than passing two different parameters to the same method.